packages/shared/src/__tests__/contracts.test.ts

```diff
@@ -37,7 +37,7 @@ describe(checkAllowance, () => {
 
     const result = await checkAllowance(spend, spender, '0x0', signer);
 
-    expect(result.isAllowable).toBe(expected);
+    expect(result.hasSufficientAllowance).toBe(expected);
```

Comment:
i was always confused by `isAllowable` 😅 this is not exposed anywhere so we can change it